_layouts/base.html

```diff
@@ -51,6 +51,7 @@
   <link rel="stylesheet" href="/css/styles.css" />
   <link rel="shortcut icon" type="image/png" href="https://assets.ubuntu.com/v1/d53eeeac-MicroK8s_favicon_64px_v2.png">
 
+  <script src="/js/build/lazysizes/lazysizes.min.js"></script>
```

Comment:
```suggestion
  <script src="/js/build/lazysizes/lazysizes.min.js" defer></script>
```

Reply:
Can you test this with `defer` and see if improves perf further? I think it should?

Reply:
I had assumed that adding `defer` would mean it would execute after the iframe script had already executed, but I just tested it, and it doesn't. Updated!